twindb_backup/modifiers/base.py

```diff
@@ -0,0 +1,41 @@
+from contextlib import contextmanager
+
+
+class ModifierException(Exception):
+    """Base Exception for Modifier error"""
+
+
+class Modifier(object):
```

Comment:
it seems quite possible a user may want to create an instance on the base class. For example, to call some callback function, but not to modify the stream